distribution/pom.xml

```diff
@@ -96,6 +96,8 @@
                                 <argument>io.druid.extensions:mysql-metadata-storage</argument>
                                 <argument>-c</argument>
                                 <argument>io.druid.extensions:postgresql-metadata-storage</argument>
+                                <argument>-c</argument>
+                                <argument>io.druid.extensions.contrib:scan-query</argument>
```

Comment:
looking through this review, I think the scan query is better than select in almost every way and I think maybe this should actually be a part of core, and be select query v2, with the goal to eventually replace the select query

Alternatively, we can keep this as an extension until we figure out the API

@gianm thoughts?